util/CMakeLists.txt

```diff
@@ -11,24 +11,41 @@ add_library(util STATIC
     src/hex_tools.cpp
     src/OpenTracing.cpp)
 
+add_library(util_shared SHARED
```

Comment:
Define a list of files to be compiled once and then use them in both static/shared add_library(...)

Reply:
Done